datadog_lambda/wrapper.py

```diff
@@ -190,6 +190,10 @@ def _after(self, event, context):
             status_code = extract_http_status_code_tag(self.trigger_tags, self.response)
             if status_code:
                 self.trigger_tags["http.status_code"] = status_code
+                if len(status_code) == 3 and status_code.startswith("5"):
+                    submit_errors_metric(context)
+                    if self.span:
+                        self.span.set_traceback()
```

Comment:
I'm not sure you have a valid stacktrace in this case, since there isn't a real python exception being thrown. I suspect you need to directly set the error message and type fields https://github.com/DataDog/dd-trace-py/blob/fb8dfa2f33fff37d21df9728d8386c0260df9744/ddtrace/contrib/grpc/server_interceptor.py#L41-L42

We should set both fields to something meaningful that can help users understand the problem when seeing it. For error.type, we can use `StatusCode 5xx`, for error.msg we can probably say `Lambda invocation returns status code 500` (we can probably spell out the actual status code in the message instead of 5xx).

@brettlangdon we are trying to mark Lambda invocations returning statusCode 5xx as errors in trace, is what I mentioned above the best way to do it?

Reply:
This is what we do in the tracer: https://github.com/DataDog/dd-trace-py/blob/3b91b0da8/ddtrace/contrib/trace_utils.py#L274-L282

tl;dr; you just need to do `self.span.error = 1`